kbase-extension/scss/modules/_variables.scss

```diff
@@ -3,7 +3,8 @@
 // Typefaces
 
 $typeface-page-text: Oxygen, Arial, sans-serif;
-$typeface-display: RobotoBlack, Arial, sans-serif;
+$typeface-display: Oxygen, Arial, sans-serif;
+$typeface-display-legacy: RobotoBlack, Arial, sans-serif;
```

Comment:
Using `$typeface-display-legacy` to refer to the old display typeface, and `$typeface-display` in places where we have replaced it with the new display typeface -- enables us to maintain the difference between display and page text typefaces in case we want to change either in the future.